### PR TITLE
Add support for GP108 (nv138)

### PR DESCRIPTION
**Pull request by Echelon9**
Adds support to envytools for the GP108 (nv138) vbios, as found in the marketing-named GT 1030.

Verified one two separate vbios seen in the wild.

### CODE REVIEW

**Comment by Echelon9** [comment]
Updated with an addition to `docs/hw/pciid.rst`

File: docs/hw/pciid.rst [hunk 1]
```diff
@@ -161,6 +161,7 @@ device id         product
 ``0x1380-0x13bf`` :ref:`GM107 <pci-ids-gm107>`
 ``0x13c0-0x13ff`` :ref:`GM204 <pci-ids-gm204>`
 ``0x1400-0x143f`` :ref:`GM206 <pci-ids-gm206>`
+``0x1d01-      `` :ref:`GP108 <pci-ids-gp108>`
```
Comment (koriakin) [inline docs/hw/pciid.rst]: 0x1d00-0x1d3f

File: docs/hw/gpu.rst [hunk 2]
```diff
@@ -698,6 +698,7 @@ unk?  \???? 0x132 GP102  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?
 unk?  \???? 0x134 GP104  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?     ?      ?   ?   ?   ?
 unk?  \???? 0x136 GP106  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?     ?      ?   ?   ?   ?
 unk?  \???? 0x137 GP107  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?     ?      ?   ?   ?   ?
+unk?  \???? 0x138 GP108  ?    ?    ?     ?   ?      ?      ?     ?    ?    ?     ?      ?   ?   ?   ?
```
Comment (koriakin) [inline docs/hw/gpu.rst]: Add the pci id here as well.  If this GPU has a HDA device, fill it in; otherwise, write \- here.

**Comment by Echelon9** [comment]
@koriakin Updated the pci-id range for the GP108 and the HDA device pci id.

In the process, I've noticed there are a number of other simple updates that could be made to complete the Maxwell and Pascal families consistently, although that will be a separate PR.

**Pull request merged**